qcportal/qcportal/manager_client.py

```diff
@@ -49,8 +49,13 @@ def __init__(
         self.manager_name_data = name_data
 
     def _update_on_server(self, manager_update: ManagerUpdateBody) -> None:
+        # Don't allow retries - we handle that elsewhere in the manager (by trying again later)
         return self.make_request(
-            "patch", f"compute/v1/managers/{self.manager_name_data.fullname}", None, body=manager_update
+            "patch",
+            f"compute/v1/managers/{self.manager_name_data.fullname}",
+            None,
+            body=manager_update,
+            allow_retries=False,
```

Comment:
Why disable retries here? I think it makes sense to have layers of retry logic; this handles the case of simple network failures, which can happen pretty often even in the best case.

Reply:
The managers have their own retry logic that works on a longer timeframe, where a manager self-destructs if there have been too many missed contacts. Right now that logic depends on knowing how many heartbeats have been missed, and adding additional time here would throw that off (not by too much in production, but enough to make testing more difficult)

Reply:
Ah cool, thanks! Makes sense here in testing; in production probably helps some with resilience for managers if the server blips for a bit.